### PR TITLE
feat: blog add sense-of-open-day

### LINKED ISSUE #44: blog: add KusionStack 开源有感
## Enhancement

https://mp.weixin.qq.com/s/5BSh4Rtf2iGEakUFIPKcGg

### PR DESCRIPTION
**Pull request by chai2010**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [X] Y fix #44  

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [X] Other: new post

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [X] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: blog/2022-06-07-sense-of-open-day/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+slug: 2022-sense-of-open-day
+title: KusionStack 开源有感
+authors:
+  name: 朵晓东
+  title: Kusion创始人
```
Comment (howieyuen) [inline blog/2022-06-07-sense-of-open-day/index.md:6]:
```suggestion
  title: Kusion 创始人
```

File: blog/2022-06-07-sense-of-open-day/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## 5. 突破与进展
+
+略过中间的种种探索和实践，回顾这段历程，在这一年多的时间里我们结合了编译技术、运维及平台技术，成功建立了一个基于 Kusion 可编程技术栈的运维体系。
+
+在业务场景上，项目覆盖了从 IaaS 到 SaaS 的大量运维场景，截至目前共接入了 800+ 应用，覆盖 9 个 BG，21 个 BU，其中典型案例交付运维提效 90% 以上，这也是蚂蚁内部第一次将大量异构应用纳入到一整套运维技术栈。
+
+在蚂蚁我们基于云原生容器和微服务技术深入探索了 DevOps、CICD 实践，完善了蚂蚁的云原生技术体系，逐步释放了云原生效率红利，同时形成了一个近 300 人的虚拟运维研发团队。
+
+不同职能不同团队的参与者凝聚在一起解决各自所面对的问题，贡献了 3W+ commit 和 35W+ 行代码，有一些参与者自发成为 Kusion 的研发者 。我认为这些工程师文化理念和领域知识的积累带来了远超运维业务本身的价值。
```
Comment (howieyuen) [inline blog/2022-06-07-sense-of-open-day/index.md:72]:
```suggestion
不同职能不同团队的参与者凝聚在一起解决各自所面对的问题，贡献了 30k+ commit 和 350k+ 行代码，有一些参与者自发成为 Kusion 的研发者 。我认为这些工程师文化理念和领域知识的积累带来了远超运维业务本身的价值。
```

File: blog/2022-06-07-sense-of-open-day/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+在历经了一段内部探索之后，我们希望把 KusionStack 开源到技术社区。因为我们意识到自身面对的问题，其他公司、团队其实也同样正在面对。借助开源这件事，我们希望团队的这些工作成果能对更多人有所帮助。
+
+当然，也受限于自身能力以及精力和资源的投入，我们希望能有更多朋友参与进来，与我们共同去完善 KusionStack，不论你是工作在云原生、运维自动化、编程语言或者是编译器中的哪一个领域，我们都非常期待和欢迎你的加入。
+
+## 7. 期待与你共成长
+
+这段经历对我来说异常宝贵，不仅仅是在于自身再一次在新的技术领域和蚂蚁的技术升级方面尝试了新的探索并实现了突破，更宝贵的是，自己还拥有了一段与一群人均 95 后的小伙伴一起将想法落地实现的奇幻历程。
+
+在未来， Kusion 的朋友圈不再局限于蚂蚁内部，面向开源，我们期待着能有更多的社区朋友在 KusionStack 与我们共同成长！
```
Comment (howieyuen) [inline blog/2022-06-07-sense-of-open-day/index.md:94]:
```suggestion
在未来，Kusion 的朋友圈不再局限于蚂蚁内部，面向开源，我们期待着能有更多的社区朋友在 KusionStack 与我们共同成长！
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: blog/2022-06-07-sense-of-open-day/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+## 3. 漫长的学习、探索与实践
+
+隔行如隔山。
+
+没有过语言设计研发的经验，也没有过开放自动化系统设计的经验，项目开展之初，我们就陷入了举步维艰的困境。
+
+经历了一段漫长时间的学习、摸索和实践的反复循环之后，项目依旧没有大的起色，更困难的是我们不但要面对蚂蚁内部复杂又耦合的场景和问题，还要经受「这种高度工程化的方式在蚂蚁是否有生存土壤」的质疑。
+
+
```
[the remark this answers is not in the document]
Comment (chai2010) [reply blog/2022-06-07-sense-of-open-day/index.md]: fixed

File: blog/2022-06-07-sense-of-open-day/index.md (hunk 1 above)
Comment (chai2010) [reply blog/2022-06-07-sense-of-open-day/index.md]: fixed

File: blog/2022-06-07-sense-of-open-day/index.md (hunk 2 above)
Comment (chai2010) [reply blog/2022-06-07-sense-of-open-day/index.md]: fixed

**Review by howieyuen — APPROVED** [review]
LGTM

**Pull request merged**